plugins/database/quota/src/main/java/org/apache/cloudstack/api/command/QuotaTariffCreateCmd.java

```diff
@@ -66,7 +66,7 @@ public class QuotaTariffCreateCmd extends BaseCmd {
             + " endDate=2009-06-03.")
     private Date endDate;
 
-    @Parameter(name = ApiConstants.POSITION, type = CommandType.INTEGER, description = "Position in the execution sequence for tariffs of the same type", since = "4.19.0.0")
+    @Parameter(name = ApiConstants.POSITION, type = CommandType.INTEGER, description = "Position in the execution sequence for tariffs of the same type", since = "4.20.0.0")
```

Comment:
should we start using three digit numbers?
```suggestion
    @Parameter(name = ApiConstants.POSITION, type = CommandType.INTEGER, description = "Position in the execution sequence for tariffs of the same type", since = "20.0.0")
```

Reply:
I don't think we need a vote for a change like this. The discussion did not lead to technical impediments. It is not forbidden to have a vote so since you ask for it, I will start one.